src/components/pages/Data.tsx

```diff
@@ -1,164 +1,152 @@
-              onClick={() => clickLink("LancetID")}
-              rel="noopener noreferrer"
-              href="https://www.thelancet.com/journals/laninf/article/PIIS1473-3099(20)30631-9/fulltext#%20"
-            >
-              <i>{Translate("LancetID")}</i>
-            </a>
-            .
-          </p>
+        <div
+          className={
+            isMobileDeviceOrTablet ? "static-mobile" : "static-content"
+          }
+        >
           <h1>{Translate("OurData")}</h1>
-          <p>{Translate("OurDataText", ["Text"])}</p>
-          <p>
-            {Translate(
-              "DataDictionaryContent",
-              ["DataDictionaryContentParagraphOne"],
-              null,
-              [false, true]
-            )}
-            <a
-              rel="noopener noreferrer"
-              onClick={() => clickLink("Manuscript Appendix")}
-              target="_blank"
-              href="https://drive.google.com/file/d/1d8-U0NgjVBTDzdj3rbAYfZyYVp0HRhgj/view?usp=sharing"
-            >
-              {Translate(
-                "ManuscriptAppendix",
-                ["ManuscriptAppendixParagraphOne"],
-                null,
-                [false, true]
-              )}
-            </a>
-            {Translate(
-              "ManuscriptAppendix",
-              ["ManuscriptAppendixParagraphTwo"],
-              null,
-              [false, true]
-            )}
-            {Translate(
-              "DataDictionaryContent",
-              ["DataDictionaryContentParagraphTwo"],
-              null,
-              [false, true]
-            )}
-            <a
-              rel="noopener noreferrer"
-              onClick={() => clickLink("Data Dictionary")}
-              target="_blank"
-              href="https://docs.google.com/spreadsheets/d/1KQbp5T9Cq_HnNpmBTWY1iKs6Etu1-qJcnhdJ5eyw7N8/edit?usp=sharing"
-            >
-              {Translate("DataDictionary")}
-            </a>
-            .
-          </p>
-          <p>
-            {Translate("DataDownload", null, null, [false, true])}
-            <Link to="/About">{Translate("SeroTrackerTeam")}</Link>.
-          </p>
+          <p>{Translate("WhatWeDoText", ["FirstParagraph"])}</p>
+          <p>{Translate("WhatWeDoText", ["SecondParagraph"])}</p>
+          <br></br>
+          <DataButtons />
+          <br></br>
+          <DataDropdowns />
+          <br></br>
+          <h1 className="mb-0">{Translate("ReferencesTable")}</h1>
         </div>
-
         <div
           className={
-            isMobileDeviceOrTablet ? "pb-3 pt-3" : "pb-3 pt-3 reference-table"
+            isMobileDeviceOrTablet
+              ? "pb-3 pt-3 static-mobile"
+              : "pb-3 pt-3 static-content"
           }
         >
           <Segment>
-            <Loader indeterminate active={isLoading} />
-            <StudiesTable
-              dataRecords={allRecords}
-              showAllStudies={false}
-            ></StudiesTable>
+            <iframe
```

Comment:
We should adjust the airtable Iframe when on mobile, it looks a bit off

Reply:
![image](https://user-images.githubusercontent.com/21212898/118372199-66cab700-b57e-11eb-8c0f-aaf40cb965b0.png)

Reply:
hmm okay so I visited the airtable view on mobile and it looks pretty much the same so not sure we can make a usability improvment here as long as we stick with the embed.

this is fine for now then